packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -839,6 +864,7 @@ async function processRequest(params) {
       },
     };
     await reconciliationReportModel.update({ name: reportRecord.name }, updates);
+    throw error;
```

Comment:
If the lambda fails, what happens to the AsyncOperation?  Does it retry the task or just fail the AsyncOperation?  Can we test it out?